### PR TITLE
Added context to logger calls

### PR DESCRIPTION
**Pull request by techsmyth**

### CODE REVIEW

File: src/utils/logging/logging-framework.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export enum LogContexts {
```
Comment (valentinyanakiev) [inline src/utils/logging/logging-framework.ts:1]: Move this enums in the winston.config.ts - this is just a config utility.

File: src/utils/data-management/data-management.service.ts [hunk 2]
```diff
@@ -240,7 +241,7 @@ export class DataManagementService {
       await this.connection.synchronize();
       this.addLogMsg(msgs, '.....dropped. Completed successfully.');
     } catch (error) {
-      this.logger.verbose(error.message);
+      this.logger.verbose(error.message, LogContexts.DATA_MGMT);
```
Comment (valentinyanakiev) [inline src/utils/data-management/data-management.service.ts:244]: Check error messages - I think there are 3 parameters for this.logger.error and the second one is not the context.

**Review by valentinyanakiev — CHANGES_REQUESTED** [review]
Move the enum to winston.config.ts and verify this.logger.error(message, context) - I think it is this.logger.error(message,undefined,context)

File: src/utils/data-management/data-management.service.ts (hunk 2 above)
Comment (techsmyth) [reply src/utils/data-management/data-management.service.ts:244]: yes but this is a verbose message

**Review by valentinyanakiev — APPROVED** [review]
👍

**Pull request merged**